manifests/kube-state-metrics-deployment.yaml

```diff
@@ -25,8 +25,6 @@ spec:
         - --telemetry-port=8082
         image: quay.io/coreos/kube-state-metrics:v1.9.7
         name: kube-state-metrics
-        securityContext:
```

Comment:
why this was removed? Now our production does not work because of this

Reply:
@zetaab Hey you can always override and use master if that works for you or release-2.0. This is just in line with what is the release-1.9 kube-state-metrics branch, feel free to raise the PR fix there against the release-1.9 branch.

Reply:
👍  we did override for this to our jsonnet